fusor-ember-cli/app/routes/deployment.js

```diff
@@ -43,6 +44,62 @@ export default Ember.Route.extend(DeploymentRouteMixin, {
 
   },
 
+  loadOpenStack(controller, model) {
+    var self = this;
+    if (model.get('deploy_openstack') && !Ember.isBlank(model.get('openstack_undercloud_password'))) {
```

Comment:
@cfchase, ok but the idea is that the validations are left to right. For example, if the deployment is valid and you go back to step 1 and erase the deployment name, then you'll notice that all the tabs will become disabled. Same thing if you make a field in the RHEV tab invalid, then all fields to the right are disabled. 

This is what I was referring to, but since you are referring to model() hooks and not just disabling/enabling, I can understand what you want to have the info loaded.